### ISSUE TITLE
Use UNIQUE_RUNE instead of a list of runes

### ISSUE THREAD

**Issue by MattTheCuber** [Altpapier/SkyHelper-Networth#51]
Most expensive runes use the id `UNIQUE_RUNE` instead of just `RUNE`. We could reduce the maintainence requirement for our list if we switch to using this in addition to a list.

## Examples
![image](https://github.com/user-attachments/assets/d2568e02-6905-483d-8089-b498603cc452)

![image](https://github.com/user-attachments/assets/8722db8e-706d-4bf9-9d33-54cea1530621)

![image](https://github.com/user-attachments/assets/f64ba7d9-50f5-49d4-b44c-2d3c23d3eeaa)

![image](https://github.com/user-attachments/assets/13f1395c-f279-4834-99da-4ff9ab203cc3)

![image](https://github.com/user-attachments/assets/d5913cab-c1b0-4913-b8c6-cf0b18194c40)

**Comment by MattTheCuber** [comment]
@DuckySoLucky is this done in #46?

**Comment by DuckySoLucky** [comment]
> [@DuckySoLucky](https://github.com/DuckySoLucky) is this done in [#46](https://github.com/Altpapier/SkyHelper-Networth/pull/46)?

Yeah, https://github.com/Altpapier/SkyHelper-Networth/blob/feat/ItemHandlerClass/calculators/helpers/SkyBlockItemNetworthHelper.js#L134-L143. We don't have rune constants anymore
